ci/gpu/build.sh

```diff
@@ -58,7 +58,8 @@ gpuci_conda_retry install -c conda-forge -c rapidsai -c rapidsai-nightly -c nvid
       "xgboost=1.3.3dev.rapidsai${MINOR_VERSION}" \
       "rapids-build-env=${MINOR_VERSION}.*" \
       "rapids-notebook-env=${MINOR_VERSION}.*" \
-      "rapids-doc-env=${MINOR_VERSION}.*"
+      "rapids-doc-env=${MINOR_VERSION}.*" \
+      "shap>=0.37,<=0.39"
```

Comment:
This will include version 0.39 as well. Did you want to exclude 0.39 or include it @dantegd

Reply:
Include it, the original had the wrong blank space because I accidentally replaced the `<=` for ` <` in a bulk replace when I was doing something else, thanks for the notice!